client/app/scripts/components/node-details.js

```diff
@@ -192,7 +192,11 @@ class NodeDetails extends React.Component {
         <div className="node-details-content">
           {details.metrics && <div className="node-details-content-section">
             <div className="node-details-content-section-header">Status</div>
-            <NodeDetailsHealth metrics={details.metrics} />
+            <NodeDetailsHealth
+              metrics={details.metrics}
+              topologyId={topologyId}
+              nodeColor={nodeColor}
```

Comment:
I would not use the node color to draw attention here. The node color distinguishes a node from other nodes. Here we merely want to highlight availability of more details. That should be distinct in its decoration.